app/templates/src/main/resources/config/liquibase/changelog/_initial_schema.xml

```diff
@@ -173,6 +173,36 @@
                                  constraintName="fk_evt_pers_audit_evt_data"
                                  referencedColumnNames="event_id"
                                  referencedTableName="jhi_persistent_audit_event"/>
+    <% if (enableSocialSignIn) { %>
+    <!-- Social -->
+    <createTable tableName="jhi_UserConnection">
+          <column name="userId" type="varchar(255)">
+              <constraints nullable="false"/>
+          </column>
+          <column name="providerId" type="varchar(255)">
+              <constraints nullable="false"/>
+          </column>
+          <column name="providerUserId" type="varchar(255)">
+              <constraints nullable="false"/>
+          </column>
+          <column name="rank" type="bigint">
+              <constraints nullable="false"/>
+          </column>
+          <column name="displayName" type="varchar(255)"/>
+          <column name="profileUrl" type="varchar(255)"/>
+          <column name="imageUrl" type="varchar(255)"/>
+          <column name="accessToken" type="varchar(255)">
+              <constraints nullable="false"/>
+          </column>
+          <column name="secret" type="varchar(255)"/>
+          <column name="refreshToken" type="varchar(255)"/>
+          <column name="expireTime" type="bigint"/>
+    </createTable>
+
+    <addPrimaryKey tableName="JHI_UserConnection" columnNames="userId, providerId, providerUserId"/>
```

Comment:
lower case here also -> some databases are case-sensitive, this would cause issues